CMakeLists.txt

```
@@ -152,8 +152,22 @@ function(configure_kernel kernel)
 endfunction()
 
 message("Configure kernels: ...")
-configure_kernel("/share/jupyter/kernels/xcpp17/")
-configure_kernel("/share/jupyter/kernels/xcpp20/")
+if(NOT EMSCRIPTEN)
```

```suggestion
if(EMSCRIPTEN)
```

please reverse the logic to read easier